data/types/bolus/normal/normal.go

```diff
@@ -9,10 +9,18 @@ import (
 const (
 	SubType = "normal" // TODO: Rename Type to "bolus/normal"; remove SubType
 
-	NormalMaximum = 100.0
-	NormalMinimum = 0.0
+	BiphasicSubType = "biphasic"
```

Comment:
For this file, see comment above on the biphasic.go file.

Reply:
same reply ;-)